core/src/errors.rs

```diff
@@ -158,6 +158,12 @@ pub enum BridgeError {
     KickoffOutpointsNotFound,
     #[error("DepositInfoNotFound")]
     DepositInfoNotFound,
+
+    #[error("FromHexError: {0}")]
+    FromHexError(hex::FromHexError),
```

Comment:
You don't have to manually implement `From` if you use this syntax.
```suggestion
    FromHexError(#[from] hex::FromHexError),
```